lib/ebaze/auctions/auction.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Ebaze.Auctions.Auction do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "auctions" do
+    field :description, :string
+    field :end_time, :utc_datetime
+    field :initial_price, :decimal
+    field :name, :string
+    field :photo_url, :string
+    field :sold_status, :boolean, default: false
```

Comment:
would just `sold` be better name since it's a yes/no type of thing rather than like a series of options?

Reply:
55f3d6f71eeb432810fefae5ec3902a76a216210

Reply:
60c7b7ff3ee7663d59279ca1dcb0c07dec3eb8bc

Reply:
agreed @ssunday, also languages have different conventions for naming booleans, this link talks more about naming booleans in Elixir: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Naming%20Conventions.md#trailing-question-mark-foo